components/card/metadata/card.yml

```
@@ -32,10 +33,11 @@ examples:
               </div>
             </div>
             <div class="spectrum-Card-content">
-              <div class="spectrum-Card-subtitle spectrum-Detail spectrum-Detail--sizeS">jpg</div>
+              <div class="spectrum-Card-description">Optional description should be kept to one or two lines</div>
```

```suggestion
              <div class="spectrum-Card-description">Optional description; should be kept to one or two lines</div>
```

Minor nit - I was just thinking these are 2 distinct notes (that description is optional and that it should be 1 or 2 lines).